planetiler-custommap/README.md

```diff
@@ -240,7 +240,17 @@ A feature is a defined set of objects that meet a specified filter criteria.
   expression should be skipped. If unspecified, no exclusion filter is applied.
 - `min_zoom` - An [Expression](#expression) that returns the minimum zoom to render this feature at.
 - `min_size` - An [Expression](#expression) that returns the minimum length of line features or square root of the
-  minimum area of polygon features to emit below the maximum zoom-level of the map.
+  minimum area of polygon features to emit below the maximum zoom-level of the map. This value is ignored if the layer
+  [Tile Post Process](#tile-post-process) is defined.
+- `min_size_at_max_zoom` - An [Expression](#expression) that returns the minimum length of line features or square root of the
+  minimum area of polygon features to emit at the maximum zoom-level of the map. This value is ignored if the layer
+  [Tile Post Process](#tile-post-process) is defined.
+- `tolerance` - An [Expression](#expression) that returns the value for the tile pixel tolerance to use when
+  simplifying features below the maximum zoom level of the map.  This value is ignored for lines or polygons if the layer
+  [Tile Post Process](#tile-post-process) `tolerance` is defined for `merge_line_strings` or `merge_polygons`, respectively.
+- `tolerance_at_max_zoom` - An [Expression](#expression) that returns the value for the tile pixel tolerance to use when
+  simplifying features at the maximum zoom level of the map. This value is ignored for lines or polygons if the layer
+  [Tile Post Process](#tile-post-process) `tolerance_at_max_zoom` is defined for `merge_line_strings` or `merge_polygons`, respectively.
```

Comment:
Can you add these to the json schema definition as well?

Reply:
See https://github.com/onthegomap/planetiler/pull/1307/commits/d2be1f53ee1252a02a654dd9911c9730954fbb13